### DISCUSSION TITLE
Converting TIFF format to CALS

### DISCUSSION THREAD

**Discussion by Zekeriya67** [ImageMagick/ImageMagick6#210] [General]
### ImageMagick version

6.9.12-72 Q8 X64

### Operating system, version and so on

Windows 10

### Description

Hello,

I try to convert a tiff files to CALS format with this  command 👍 

`convert image.tif -compress Group4 -format CALS image.cal`

But when I see the options with an viewer on my image.cal file, it is said that is a TIFF file. What I may wrong ?

**Comment by snibgo** [comment]
First, check that your variety of IM can write CALCS files:
```
C:\web\im>%IMG7%magick -list format |grep -i cals

     CALS* rw-   Continuous Acquisition and Life-cycle Support Type 1
```
My variety has "w", so it can write CALS.

Your "-format CALS" has no effect. That would set the format for "info:", which you are not using.

Try the conversion:
```
C:\web\im>%IMG7%magick out.tiff -compress Group4 x.cals
```
Check what IM says about x.cals:
```
C:\web\im>%IMG7%magick x.cals info:
x.cals CALS 139x162 139x162+0+0 1-bit Bilevel Gray 911B 0.000u 0:00.000
```
It seems to have worked.

Be aware that not all file formats can use all compression methods.

**Reply by Zekeriya67** [reply]
Which version of IM do you use ? Does it works on Windows ?

**Comment by snibgo** [comment]
I used IM v7.1.0-49 on Windows 11.

IM v6 is now very old. I do not recommend using it, except for compatibility with old scripts.

**Comment by Zekeriya67** [comment]
Thank you it works great !
